Tasks/Common/AzureRmDeploy-common/operations/ContainerBasedDeploymentUtility.ts

```diff
@@ -203,4 +224,56 @@ export class ContainerBasedDeploymentUtility {
         delete webAppSettings["properties"]["DOCKER_REGISTRY_SERVER_USERNAME"];
         delete webAppSettings["properties"]["DOCKER_REGISTRY_SERVER_PASSWORD"];
     }
+
+    private updateImagesInConfigFile(configFilePath, images): string {
+        const tempDirectory = this.getTempDirectory();
+        var contents = fs.readFileSync(configFilePath).toString();
+        var imageList = images.split("\n");
+        imageList.forEach((image: string) => {
+            let imageName = image.split(":")[0];
+            if (contents.indexOf(imageName) > 0) {
+                contents = this.tokenizeImages(contents, imageName, image);
+            }
+        });
+
+        let newFilePath = path.join(tempDirectory, path.basename(configFilePath));
+        fs.writeFileSync(
+            path.join(newFilePath),
+            contents
+        );
+        
+        return newFilePath;
+    }
+
+    private getTempDirectory(): string {
```

Comment:
Move this function to azure-pipelines-tasks/Tasks/Common/AzureRmDeploy-common/webdeployment-common/utility.ts

Reply:
it is done.